src/types/fieldExtractor.ts

```diff
@@ -81,3 +134,43 @@ export const resolveEthereumTrackIdOverrides = (nft: NFT, contract: NftFactory):
   }
   return ethereumTrackId(nft.contractAddress, trackId);
 }
+
+export const resolveArtworkUrlOverrides = (nft: NFT, contract: NftFactory): string => {
+  const override = contract.typeMetadata?.overrides?.extractor?.artworkUrl;
+
+  if (override === ArtworkUrlExtractorTypes.USE_ARTWORK_URL_OVERRIDE) {
+    const track = contract.typeMetadata?.overrides?.track;
+    const url = (track as any).lossyArtworkURL;
+    if (!url) {
+      throw new Error('track lossy artwork url override not provided');
+    }
+    return url;
+  }
+  return nft.metadata.image
+}
+
+export const resolveArtistNameOverrides = (nft: NFT, contract: NftFactory): string => {
```

Comment:
i don't see where overrides.artist.name is being used ever? is it no longer used? eg: on danial allen, we specify overrides.artist.name but never specify overrides.extractor and so it doesn't seem like the name will ever be used. did you try recreate daniel allen/check this? i may be missing something, but at the moment it looks like the name will be wrong if we recreate?

Reply:
the resolveArtistIdOverrides logic looks good though - should this one look more like that?

Reply:
They are still being used as the highest priority override by being destructured, eg in the `mapArtistProfile` method:
```ts
return {
    // ...
    createdAtEthereumBlockNumber: nft.createdAtEthereumBlockNumber,
    ...contract.typeMetadata?.overrides?.artist
}
```

Reply:
Given that we're shifting the fallback for artist id to be `ethereumArtistId(nftFactoryAddress)`, I'll make an explicit `ArtistIdExtractorTypes` ->`USE_ARTIST_ID_OVERRIDE = 'useArtistIdOverride',` and use it explicitly where needed.

Reply:
> the resolveArtistIdOverrides logic flow looks good though - should this one look more like that?

I've applied a similar strategy for ArtistName and removed the fallback to platformId as I think it isn't a good default. You now have to explicitly specify how to deal with the artist name